core-bundle/src/Routing/RouteProvider.php

```diff
@@ -421,7 +421,7 @@ private function sortRoutes(array &$routes, array $languages = null): void
                         return -1;
                     }
 
-                    return $pageB->rootIsFallback ? 1 : 0;
+                    return $pageB->rootIsFallback ? 1 : strcmp($pageA->rootSorting, $pageB->rootSorting);
```

Comment:
This will cause a `TypeError: strcmp() expects parameter 1 to be string, null given` error if one of the values is null (see https://travis-ci.com/contao/contao/jobs/174317669#L501). Do you want me to cast the arguments?

Reply:
how can that value ever be `null`?

Reply:
Probably only in the unit tests?

Reply:
Probably? 😂 I guess casting won't harm… but then should we also cast all other (e.g. alias at the bottom of this function)?

Reply:
Fixed in 6fcd4ee